extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts

```diff
@@ -124,17 +124,17 @@ export class InstanceSummary {
 
 		const labels = [
 			{
-				label: "Not ready",
+				label: constants.NOT_READY,
 				count: 1,
 				color: "#E00B1C"
```

Comment:
can these colors be replaced with named constants that reflect the color or purpose

Reply:
done